packages/terra-application-menu-layout/src/ApplicationMenuLayout.jsx

```diff
@@ -0,0 +1,107 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import classNames from 'classnames/bind';
+import 'terra-base/lib/baseStyles';
+import AppDelegate from 'terra-app-delegate';
+
+import styles from './ApplicationMenuLayout.scss';
+
+const cx = classNames.bind(styles);
+
+const propTypes = {
+  /**
+   * The AppDelegate instance provided by the containing component. If present, its properties will propagate to the children components.
+   * */
+  app: AppDelegate.propType,
+  /**
+   * Content element to be placed within the fill area of the header.
+   * */
+  content: PropTypes.element,
+  /**
+   * Extensions element to be placed before the end of the header.
+   * */
+  extensions: PropTypes.element,
+  /**
+   * Footer element to be placed at the end of the header.
+   * */
+  footer: PropTypes.element,
+  /**
+   * Header element to be placed at the start of the header.
+   * */
+  header: PropTypes.element,
+  /**
+   * Layout configuration to handle size and toggle functionality.
+   * */
+  layoutConfig: PropTypes.shape({
+    size: PropTypes.string,
+    toggleMenu: PropTypes.func,
+    menuIsOpen: PropTypes.bool,
+    togglePin: PropTypes.func,
+    menuIsPinned: PropTypes.bool,
+  }),
+};
+
+const defaultProps = {
+  layoutConfig: {
```

Comment:
Is there a default size for this? Or is the default size passed via the app delegate?